Source/Engine/Utilities/Extensions.cs

```diff
@@ -300,6 +300,18 @@ public static Vector2 NextUnitVector2(this Random random, float radius = 1.0f)
             var randomRadius = (float)random.NextDouble() * radius;
             return new Vector2((float)Math.Cos(random.NextDouble()) * randomRadius, (float)Math.Sin(random.NextDouble()) * randomRadius);
         }
+        
+        /// <summary>
+        /// Generates a random <see cref="Vector2"/> point on a circle of a given radius.
+        /// </summary>
+        /// <param name="random">An instance of <see cref="Random"/>.</param>
+        /// <param name="radius">Radius of circle. Default 1.0f./>.</param>
+        /// <returns>A random <see cref="Vector2"/>.</returns>
+        public static Vector2 NextUnitCircleVector2(this Random random, float radius = 1.0f)
+        {
+            double randomDegree = random.NextDouble() * 360;
```

Comment:
I think this should be: `double randomDegree = random.NextDouble() * Mathf.RevolutionsToRadians;` so cos/sin will get proper value in radians.

Reply:
You're right. I assumed it was degrees since the result was still what was desired. Fixed in new commit.